src/elements/LayoutAuth.tsx

```diff
@@ -1,20 +1,25 @@
 import { Box, Flex, Heading, Icon, Text } from '@chakra-ui/react'
-import { useState } from 'react'
+import { ReactNode, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { FaChevronLeft } from 'react-icons/fa'
 import { NavLink, Outlet } from 'react-router-dom'
 import AuthBanner from '~components/Organization/Dashboard/AuthBanner'
+import { History } from 'history'
 
 export type AuthOutletContextType = {
   setTitle: React.Dispatch<React.SetStateAction<string>>
   setSubTitle: React.Dispatch<React.SetStateAction<string>>
+  setBackRoute: React.Dispatch<React.SetStateAction<History.LocationState>>
+  setSidebar: React.Dispatch<React.SetStateAction<ReactNode>>
 }
 
 const LayoutAuth = () => {
   const { t } = useTranslation()
 
   const [title, setTitle] = useState('')
   const [subTitle, setSubTitle] = useState('')
+  const [sidebar, setSidebar] = useState<ReactNode>(null)
+  const [backRoute, setBackRoute] = useState<History.LocationState>('/')
```

Comment:
Rename it to `back` && `setBack` to be consistent with other layouts

Reply:
b18a0c9